bin/dipy_nlmeans

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env python
```

Comment:
@Garyfallidis @matthieudumont : I think `#!python` should be used here right?
Difference as discussed earlier:
1. `#! /usr/bin/env python` : The command can be used from a virtual environment but it cannot be used systemwide.

2. `#!python` : The command can be used systemwide. But it does not work from a virtual environment.